the Info.plist of iOS project from source code can't parse

<!-- ## IMPORTANT -->
<!-- Issues are ONLY for reporting bugs and feature requests. For support, questions, queries and discussions use our slack channel. Slack Channel: https://mobsf.slack.com/ , Get Invitation: https://goo.gl/6cYU5a
-->
## EXPLANATION OF THE ISSUE

<!-- What happens, under which versions, under what conditions, when, and what were you expecting instead. -->

## STEPS TO REPRODUCE THE ISSUE

```
Mobile-Security-Framework-MobSF/StaticAnalyzer/views/ios/plist_analysis.py:200
```
In my iOS project, the Info.plist file name is `Info.plist`

https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/AboutInformationPropertyListFiles.html
## CONSOLE OUTPUT

```
[2018-08-27 08:30:56]
[ERROR] Updating DB (/Users/mark/GitHub/downloads/Mobile-Security-Framework-MobSF/StaticAnalyzer/views/ios/db_interaction.py, LINE 248 "INFOPLIST=info_dict["plist_xml"],"): 'NoneType' object is not subscriptable

[2018-08-27 08:30:56]
[ERROR] Rendering to Template (/Users/mark/GitHub/downloads/Mobile-Security-Framework-MobSF/StaticAnalyzer/views/ios/db_interaction.py, LINE 150 "'plist': info_dict["plist_xml"],"): 'NoneType' object is not subscriptable
```

--- a/StaticAnalyzer/views/ios/plist_analysis.py
+++ b/StaticAnalyzer/views/ios/plist_analysis.py
@@ -197,7 +197,7 @@ def plist_analysis(src, is_source):
                 if ifile.endswith(".xcodeproj"):
                     app_name = ifile.replace(".xcodeproj", "")
                     break
-            app_plist_file = app_name + "-Info.plist"
+            app_plist_file = "Info.plist"
             for dirpath, dirnames, files in os.walk(src):
                 for name in files:
                     if "__MACOSX" not in dirpath and name == app_plist_file:

